airflow-core/src/airflow/ui/src/i18n/locales/de/dag.json

```
@@ -53,17 +66,45 @@
       "label": "Richtung des Graph"
     }
   },
+  "paramsFailed": "Laden der Parameter fehlgeschlagen",
+  "parse": {
+    "toaster": {
+      "error": {
+        "description": "Die Anfrage zum Parsen des Dag ist fehlgeschlagen. Eventuell gibt es schon bestehende Anfragen.",
+        "title": "Neues Parsen des Dag fehlgeschlagen"
+      },
+      "success": {
+        "description": "Das Dag sollte gleich neu geparst sein.",
+        "title": "Anfrage zu Dag parsen erfolgreich eingestellt"
```

```suggestion
        "title": "Anfrage zu Dag parsen erfolgreich beendet"
```
nur ein Stil Vorschlag :)

---

Ah, hier geht es darum dass die Anfrage gestartet wird dass das Backend es mal asynchron macht. Eingestellt klingt irgendwie auch "komisch". Ich nehme mal "gestartet" wie bei dem Vorschlag oben.